packages/features/bookings/Booker/components/Away.tsx

```diff
@@ -0,0 +1,22 @@
+import { useLocale } from "@calcom/lib/hooks/useLocale";
```

Comment:
Couldn't find a design for this, so for now I took the in current version. Will check with Ciaran.